tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -985,11 +998,177 @@ public void SetDefaultExtensions(string recordingId = null)
 
                     throw new HttpException(HttpStatusCode.BadRequest, sb.ToString());
                 }
+
+
+
+
+                // TODO
+                // - create an option in sanitizer to get access to 'ignore case' and other regex customizations
+                // - turn commented lines into actual sanitizers after discussion
+                // - add a query param sanitizer
                 Sanitizers = new List<RecordedTestSanitizer>
                 {
+                    // basic RecordedTestSanitizer handles Authorization header
                     new RecordedTestSanitizer(),
+
+                    // new defaults begin
+                    new GeneralRegexSanitizer(regex: "SharedAccessKey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    new GeneralRegexSanitizer(regex: "AccountKey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    // "containerUrl": "<URL>/<Container>?sp=...st=...se=...spr=...sv=...sr=...sig="
```

Comment:
Do we need a UriRegexSanitizer here?

Reply:
I think this one should actually be `$..containerUrl` getting sanitized out.

Looking at the common list + this master list, I don't see the SAS URI Suffix (for want of a better term) as a global sanitizer. So I'll add that.